lib/screens/app/wallet/receive.dart

```diff
@@ -539,6 +541,18 @@ class _ReceiveFormState extends State<ReceiveForm> {
                 }
               },
             ),
+            Align(
+                alignment: Alignment.topLeft,
+                child: Padding(
+                    padding: EdgeInsets.fromLTRB(16, 5, 0, 0),
+                    child: Consumer<RateNotifier>(
+                      builder: (context, rateModel, child) {
+                        return Text(
+                          '${rateModel.rate?.usdString(invoiceAmountDouble)}',
```

Comment:
Please handle rate==null case - make sure it shows the empty string in that case.